test/functional/rpc_net.py

```
@@ -7,11 +7,18 @@
 Tests correspond to code in rpc/net.cpp.
 """
 
-from test_framework.p2p import P2PInterface
+from decimal import Decimal
```

Ruff (0.12.2)

`decimal.Decimal` imported but unused

Remove unused import: `decimal.Decimal`

(F401)